common/djangoapps/django_comment_common/models.py

```diff
@@ -157,6 +158,13 @@ def all_permissions_for_user_in_course(user, course_id):  # pylint: disable=inva
             if not permission_blacked_out(course, role_names, permission.name):
                 permission_names.add(permission.name)
 
+    # Prevent a circular import
+    from django_comment_common.utils import GLOBAL_STAFF_ROLE_PERMISSIONS
```

Comment:
Is there another way to avoid the circular import? The rest of the PR looks good to me.

Reply:
Unfortunately, I was unable to find another way.

Reply:
Same question - is this how this is handled in other parts of the repository?